src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/SettingsProperty.cs

```diff
@@ -5,6 +5,8 @@ namespace System.Configuration
 {
     public class SettingsProperty
     {
+        internal static bool EnableUnsafeBinaryFormatterInPropertyValueSerialization { get; } = AppContext.TryGetSwitch("System.ComponentModel.TypeConverter.EnableUnsafeBinaryFormatterInPropertyValueSerialization", out bool isEnabled) ? isEnabled : false;
```

Comment:
It feels wierd that the switch name has System.ComponentModel.TypeConverter when the assembly in question here is System.Configuration.ConfigurationManager. I suppose this is because that switch already existed, but should we consider renamining the switch so that it applies to multiple libraries?

Reply:
ugh sorry my bad. I fixed this locally but forgot to push it. Will push this and fix the other comments  in a bit